### PR TITLE
Feature/rw about page

### PR DESCRIPTION
**Pull request by rwilliams659**
# Som Timer Pull Request Form

## Description
- About component created with all descriptive copy as well as Contributor list of each member, linked to their GitHub profile
- About styling added
- About test added to ensure all elements are rendered correctly 

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other, Please explain:
`Your message here`
## Has This Been Tested?
- [ ] No
- [x] Yes

# Message for reviewer:
Still open to any copy suggestions! I wrote this up pretty quick so we'd have something in there
# Checklist:
- [X] My code has no unused/commented out code
- [X] My code has no console logs
- [X] I have performed a self-review of my own code
- [X] I have commented on my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/Components/About/About.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+const About = () => {
+  return (
+    <section className={style.About}>
+      <section>
+        <h2>About Som Timer</h2>
+        <section className={style.appDescription}>
+          <p><b>Som Timer</b> is a timer that cares. Inspired by the Pomodoro Technique®, the practice of taking regular breaks at set intervals to fight cognitive boredom and burnout, increase productivity, and boost motivation, Som Timer takes the concept one step further. We at Som Timer believe that what you do on your breaks is just as important as the decision to take those breaks. </p>
+          <p>Mindfulness meditation and other somatic exercises have been linked to enhanced productivity, boosted mental health, and stress reduction. To encourage this state of mind and take the guesswork out of your breaks, Som Timer provides mindfulness-related content for you during your break intervals, so you can get back to work feeling refreshed, calm, and ready to tackle the next task.</p>
+          <p className={style.disclaimer}>This timer is not affiliated with, associated with, or endorsed by the Pomodoro Technique® or its creator, Francesco Cirillo.
```
Comment (jkwest-93) [inline src/Components/About/About.js:12]: I really appreciate how clear & concise this is! Great work giving all this information to our users.

**Review by jkwest-93 — APPROVED** [review]
Great work Rachel!! This reads really well.

File: src/Components/About/About.test.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    expect(frontEndHeading).toBeInTheDocument();
+    expect(frontEndEngineer1).toBeInTheDocument(); 
+    expect(frontEndEngineer2).toBeInTheDocument(); 
+    expect(frontEndEngineer3).toBeInTheDocument(); 
+    expect(backEndHeading).toBeInTheDocument();
+    expect(backEndEngineer1).toBeInTheDocument();
+    expect(backEndEngineer2).toBeInTheDocument();
+    expect(backEndEngineer3).toBeInTheDocument();
+  })
+})
```
Comment (Abdeboskey) [inline src/Components/About/About.test.js:37]: Waaaaaaoooo what a beefy test!!!!

**Review by Abdeboskey — APPROVED** [review]
This is excellent, thank you for putting this extra time in Rachel!

**Review by sciencefixion — APPROVED** [review]

**Pull request merged**